### ISSUE TITLE
gopass doesn’t release terminal after sync

### ISSUE THREAD

**Issue by walteweiss** [gopasspw/gopass#1423]
### Summary

When I type `gopass sync`, sync happens, but it does not release the terminal after it’s done, e.g. I can type anything, but nothing happens.

### Steps To Reproduce

For me it’s just `gopass sync`.

### Expected behavior

I expect I can enter commands after the sync finishes.

### Environment

- OS: Arch Linux
- OS version: `Linux arch-mba11-2010 5.7.2-arch1-1 #1 SMP PREEMPT Wed, 10 Jun 2020 20:36:24 +0000 x86_64 GNU/Linux`
- gopass Version: 

```
gopass 1.9.2 (2020-05-13 21:14:02) go1.14.2 linux amd64
<root>     - gpg 2.2.20 - git 2.27.0 -   fs 0.1.0
Available Crypto Backends: age, gpgcli, plain, xc
Available RCS Backends: gitcli, noop
Available Storage Backends: fs, inmem
```

- Installation method: [pacman](https://www.archlinux.org/packages/community/x86_64/gopass/)

### Additional context

Not sure whether all that matters here, but just in case:
- I use gpg agent instead of ssh agent,
- and I gpg-sign my commits as well,
- I use one external mount (synced with others for work), besides my root mount, both are populated with passwords,
- [attaching the screenshot](https://user-images.githubusercontent.com/59926876/84875592-d4304800-b08e-11ea-9e73-24c48c2a68d9.png),
- the terminal is [alacritty](https://github.com/alacritty/alacritty) `0.4.3`.

**Comment by dominikschulz** [comment]
Thanks for reporting this issue.

So far I couldn't reproduce it.
I've tried in alacritty and have extneral mounts and commit signing.

What do you mean by "I use gpg for ssh"? The gpg agent?
That shouldn't harm but I can reproduce that easily.

**Comment by walteweiss** [comment]
You are very welcome, thank you for helping to contribute to such a greap upgrade to classic pass! I really enjoy using `gopass`, that’s my favourite password manager so far!

This bug is very difficult to catch, right after me issuing the issue I couldn’t reproduce it myself. It happened once again just now, but I’m not aware of the circumstances that triggered it. Maybe it happened after the reboot, I’m not sure: I rebooted my laptop yesterday’s night, and I `gopass sync` this morning for the first time since then.

Yes, I mean gpg agent that is used instead of ssh agent. I believe that should make no difference, but I mentioned that just in case.

My setup is pretty minimal, basically all I use is: sway (wayland), firefox, telegram, alacritty with some tools (like ranger or vim). Not even an unusual daemon is running. So I have no idea what else to add to this. I’m attaching [the list of all my packages](https://github.com/gopasspw/gopass/files/4807417/pacman_packages_2020-06-17-105024.log), made with `pacman -Qe`.

I assume this bug would be very difficult to reproduce, so maybe that would be a wise idea to close it, but it’s all up to you guys, I just wanted to contribute this tiny way and let you know that happens once in a while.

**Comment by walteweiss** [comment]
Some extra information from me. 

This bug happens every time at the first sync after reboot. Even when my Wi-Fi isn’t connected yet, and I run `gopass sync` it gives me a standard error, and then I’m unable to use terminal until I force quit it. 

I haven’t mentioned that, but it ’catches‘ just one terminal window (I have many of them opened) and it doesn’t hang a terminal, so I cannot use other windows, it just doesn’t release the current window it was issued from.

**Comment by dominikschulz** [comment]
I'm sorry, but I can hardly believe that gopass itself is causing this. We're doing very little terminal trickery (only some for reading passwords without echo). Even less in the current master. Sorry.

Not reproducible.

**Comment by walteweiss** [comment]
Hey, excuse me for bothering you with this issue, but just to let you know that this issue is gone right after I updated to `gopass 1.10.0-rc.1 (2020-08-23 18:46:22) go1.15 linux amd64`, nothing else was updated, as yesterday I had this very bug, but today right after the update (no other packages were updated), it works as expected. 

I’m on Arch Linux, right now it’s `Linux arch-mba11-2010 5.8.3-arch1-1 #1 SMP PREEMPT Fri, 21 Aug 2020 16:54:16 +0000 x86_64 GNU/Linux`

Maybe at some point this info will make some value, who knows. 

Cheers!
Walter
